atom/browser/net/atom_network_delegate.cc

```diff
@@ -378,7 +378,7 @@ void AtomNetworkDelegate::OnResponseStarted(net::URLRequest* request,
     return;
   }
 
-  if (request->status().status() != net::URLRequestStatus::SUCCESS)
+  if (net_error != net::URLRequestStatus::SUCCESS)
```

Comment:
this is not correct, It should be net::OK

Reply:
Fixed.